kolibri/core/auth/management/commands/bulkexportusers.py

```diff
@@ -66,14 +66,14 @@
 # These constants must be entered vertbatim in the CSV
 roles_map = {
     role_kinds.ADMIN: "ADMIN",
-    role_kinds.COACH: "COACH",
-    role_kinds.ASSIGNABLE_COACH: "ASSIGNABLE_COACH",
+    role_kinds.COACH: "FACILITY_COACH",
+    role_kinds.ASSIGNABLE_COACH: "CLASS_COACH",
 }
 
 
 def not_specified(field, obj):
     val = obj[field]
-    return None if val == "NOT_SPECIFIED" else val
+    return None if (val == "NOT_SPECIFIED" or val == DEFERRED) else val
```

Comment:
Shouldn't `NOT_SPECIFIED` and `DEFERRED` be treated as separate values?

Reply:
Honestly, I don't know. Our code is not consistent on this. My impression is that `NOT_SPECIFIED` was used in the past and `DEFERRED` is being used recently. In any case, when exporting it seems that not returning any value in the cvs is a good policy because none of both will have a real meaning for the user (specially if the are not English spoken).

Reply:
they are different concepts:

* "Not specified" means the user intentionally chose to not to provide an answer
* "Deferred" means the user hasn't made a decision yet

I agree that an empty value should be a synonym for `NOT_SPECIFIED`